code/style.css

```diff
@@ -119,22 +127,87 @@ main {
   display: flex;
 }
 
+#name-input {
+  background-color: #fff; 
+  border: 1px solid grey;
+  border-radius: 1rem;
+  margin-right: 0.5rem;
+  margin-top: 10px;
+}
+
 button {
-  background-color: #0026ff;
+  background-color: #EC255A;
   color: white;
   border: none;
   border-radius: 4px;
-  padding: 16px 40px;
+  padding: 16px 30px;
   margin-right: 4px;
   font-size: 16px;
   line-height: 26px;
   font-family: 'Montserrat';
   font-weight: 500;
   cursor: pointer;
   transition: all 0.3s ease;
+  border-radius: 1rem;
+  position: relative;
+ z-index: 1;
+ overflow: hidden;
 }
 
 button:hover {
   opacity: 0.9;
   transition: all 0.2s ease;
 }
+
+button:after {
+  content: "";
+  background: white;
+  position: absolute;
+  z-index: -1;
+  left: -20%;
+  right: -20%;
+  top: 0;
+  bottom: 0;
+  transform: skewX(-45deg) scale(0, 1);
+  transition: all 0.5s;
+ }
+ 
+ 
+ 
+ button:hover:after {
+  transform: skewX(-45deg) scale(1, 1);
+  -webkit-transition: all 0.5s;
+  transition: all 0.5s;
+ }
```

Comment:
Love the hover effect on the buttons! It's very clean yet stylish and smooth!

Reply:
Thanks Mathilda for your kind words and all your comments. We have been looking into your comments together and we appreciate all of it. Best regards Nabeel & Josefin